pkg/typeutil/convension.go

```diff
@@ -34,3 +34,11 @@ func Uint64ToBytes(v uint64) []byte {
 	binary.BigEndian.PutUint64(b, v)
 	return b
 }
+
+// BoolToUint64 converts bool to uint64.
+func BoolToUint64(b bool) uint64 {
+	if b {
+		return 1
+	}
+	return 0
+}
```

Comment:
What about adding a unit test here?

Reply:
I think this is just a conditional operator.